src/main/java/de/numcodex/feasibility_gui_backend/terminology/TerminologyService.java

```diff
@@ -43,6 +45,9 @@ public class TerminologyService {
   private Map<UUID, TerminologyEntry> terminologyEntriesWithOnlyDirectChildren = new HashMap<>();
   private Map<UUID, Set<TerminologyEntry>> selectableEntriesByCategory = new HashMap<>();
 
+  @Value("classpath:de/numcodex/feasibility_gui_backend/terminology/terminology-systems.json")
```

Comment:
the terminology-systems.json should be part of the ontology an supplied as such

Reply:
it's moved to filesystem now and will also be taken from the ontology generator once done